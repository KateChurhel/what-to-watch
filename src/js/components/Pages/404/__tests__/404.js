import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import PageNotFound from '../index';

it('Page Not Found (404) correctly renders', () => {
  const tree = shallow(<PageNotFound />);
  expect(toJSON(tree)).toMatchSnapshot();
});
