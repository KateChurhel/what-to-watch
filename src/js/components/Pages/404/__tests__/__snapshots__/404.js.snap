// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page Not Found (404) correctly renders 1`] = `
<div
  className="container page-not-found"
>
  <div
    className="page-not-found__title"
  >
    404
  </div>
  <p
    className="page-not-found__subtitle"
  >
    Oops! Page not found
  </p>
  <p
    className="page-not-found__description"
  >
    Please go to the Home page by clicking the button below.
  </p>
  <button
    className="button page-not-found__button"
    onClick={[Function]}
    type="button"
  >
    Go to Home page
  </button>
</div>
`;
